static/app/views/alerts/rules/metric/ruleConditionsForm.tsx

```diff
@@ -732,6 +743,29 @@ class RuleConditionsForm extends PureComponent<Props, State> {
                         onChange(query, {});
                       }}
                     />
+                  ) : alertType === 'eap_metrics' ? (
+                    <SpanTagsProvider dataset={DiscoverDatasets.SPANS_EAP}>
```

Comment:
You may want to list this Provider higher at some point so you can use it to populate other parts of the form.

Reply:
👍 I think I'll save this for a follow up PR. Just want to keep these providers and context more localized for now until Explore is closer to EA and avoid running these providers higher up conditionally.